content/guides/embed/ui-elements/installation.md

```diff
@@ -44,7 +32,7 @@ npm install box-ui-elements
 <Message warning>
   # Content Preview
 
-The content Content Preview element is currently not available via NPM.
+The Box Content Preview library is currently not available via NPM.
```

Comment:
the element is available through NPM but the SDK is not: https://github.com/box/box-content-preview